### ISSUE TITLE
Nas search

### ISSUE THREAD

**Issue by annatelegina** [skmhrk1209/Single-Path-NAS-PyTorch#3]
Hello!
Thank you very much for your work. 
Is this the final version of the code or it will be updated?

**Comment by skmhrk1209** [comment]
Hi, thanks for your comment.
This is a work in progress, and I will push the final code with a searching result in a week.

**Comment by Abeni18** [comment]
Hello !,
It's a nice job, would you be able to finish it, is this the final version?

**Comment by skmhrk1209** [comment]
Hi, 
I implemented the module following the paper,  and works when using only cross entropy loss.
But I haven't demonstrate if the module works well using runtime loss, and I have not time to do that...
